src/functionaltest/java/com/ericsson/ei/frontend/TestRulesFunctionality.java

```diff
@@ -0,0 +1,89 @@
+package com.ericsson.ei.frontend;
+
+import static org.junit.Assert.assertEquals;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import com.ericsson.ei.config.SeleniumConfig;
+import com.ericsson.ei.frontend.pageobjects.IndexPage;
+import com.ericsson.ei.frontend.pageobjects.TestRulesPage;
+
+public class TestRulesFunctionality extends SeleniumBaseClass {
+
+    @Test
+    public void testJourneyToFindAggregatedObjectButton() throws Exception {
+        // Load index page and wait for it to load
+        IndexPage indexPageObject = new IndexPage(mockedHttpClient, driver, baseUrl);
+        indexPageObject.loadPage();
+        TimeUnit.SECONDS.sleep(3);
+
+        // Verify that we can navigate to test rules page
+        TestRulesPage testRulesPage = indexPageObject.clickTestRulesPage();
+        new WebDriverWait(driver, 10).until((webdriver) -> testRulesPage.presenceOfTestRulesHeader());
+
+        // Verify that "download rules template" button works
+        String mockedResponse = this.getJSONStringFromFile(
+                "src/functionaltest/resources/responses/RulesTemplateObject.json");
```

Comment:
All filepaths could be added as class variables for easier editing. private static final String PATH_TO_SOME_FILE = "src/...."

Reply:
DONE